addons/chemlights/CfgVehicles.hpp

```diff
@@ -0,0 +1,279 @@
+
+class CBA_Extended_EventHandlers;
+
+class CfgVehicles {
+
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class ACE_Equipment {
+                class ACE_Chemlights {
+                    displayName = CSTRING(Action_Chemlights);
+                    icon = "\a3\ui_f\data\gui\cfg\Hints\chemlights_ca.paa";
+                    condition = QUOTE((count ([ACE_player] call FUNC(getShieldComponents)) > 0) || (count ([ACE_player] call FUNC(getShields)) > 0));
+                    statement = "true";
+                    exceptions[] = {"isNotDragging", "notOnMap", "isNotInside", "isNotSitting"};
+                    insertChildren = QUOTE(_this call DFUNC(compileChemlightMenu));
+                    showDisabled = 0;
+                    priority = 99;
+                };
+            };
+        };
+    };
+    
+    class Thing;
+    class ThingX;
+    
+    class ACE_Chemlight_IR_Marker: Thing {
+        author = ECSTRING(common,ACETeam);
+        displayName = "ACE Chemlight IR Marker";
+        mapSize = 0.2;
+        scope = 2;
+        type = 1;
+        vehicleClass = "";
+        simulation = "nvmarker";
+        nvTarget = 1;
+        brightness = 0.2;
+        class NVGMarker {
+            diffuse[] = {0.1,0.1,0.1};
+            ambient[] = {0.01,0.01,0.01};
+            brightness = 0.015;
+            name = "pozicni blik";
+            drawLight = 0;
+            drawLightSize = 0;
+            drawLightCenterSize = 0;
+            activeLight = 0;
+            blinking = 0;
+            dayLight = 0;
+            onlyInNvg = 1;
+            useFlare = 0;
+        };
+    };
+    
+    class ACE_Chemlight_IR_Dummy: ThingX {
+        ACE_Attachable = "ACE_G_Chemlight_IR";
+        author = ECSTRING(common,ACETeam);
+        displayName = "ACE Chemlight IR Phys Dummy";
```

Comment:
I don't think a dummy needs a translation - if the name never appears anywhere outside of debugging. scope should be 1 in that case though